src/BlockTypes/ProductButton.php

```diff
@@ -115,7 +115,7 @@ protected function render( $attributes, $content, $block ) {
 			$context = array(
 				'woocommerce' => array(
 					'isLoading'     => false,
-					'numberOfItems' => $number_of_items_in_cart,
+					'numberOfItems' => strval( $number_of_items_in_cart ),
```

Comment:
I think there is an inconsistency between this line and L157. There we set this property to be an int:

```
$context['woocommerce']['numberOfItems'] = 0;
```